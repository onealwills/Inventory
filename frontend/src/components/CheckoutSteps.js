import React from "react";

export default function CheckoutSteps(props) {
  return (
    <div className="row checkout-steps">
      <div className={props.steps1 ? "active" : ""}>Signin</div>
      <div className={props.steps2 ? "active" : ""}>Shipping</div>
      <div className={props.steps3 ? "active" : ""}>Payment</div>
      <div className={props.steps4 ? "active" : ""}>PlaceOrder</div>
    </div>
  );
}
