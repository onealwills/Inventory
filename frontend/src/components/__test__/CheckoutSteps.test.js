import { render } from "@testing-library/react";
import CheckoutSteps from "../CheckoutSteps";

test("<CheckoutSteps/> component snapshot test", () => {
  const component = render(<CheckoutSteps step1 step2></CheckoutSteps>);
  expect(component.container).toMatchSnapshot();
});
