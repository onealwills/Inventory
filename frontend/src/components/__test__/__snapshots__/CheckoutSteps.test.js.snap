// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CheckoutSteps/> component snapshot test 1`] = `
<div>
  <div
    class="row checkout-steps"
  >
    <div
      class=""
    >
      Signin
    </div>
    <div
      class=""
    >
      Shipping
    </div>
    <div
      class=""
    >
      Payment
    </div>
    <div
      class=""
    >
      PlaceOrder
    </div>
  </div>
</div>
`;
